util/symbolize/dwarf3ranges.cc

```diff
@@ -63,10 +63,12 @@ void AddressRangeList::ReadDwarfRangeList(uint64 offset, uint64 base,
 
 void AddressRangeList::ReadDwarfRngListsDirectly(uint64 offset, uint64 base,
                                             AddressRangeList::RangeList* ranges, uint64 addr_base) {
-  // When we are reading with DW_FORM_sec_offset, the .debug_rnglists
-  // does not care about which RngListsData we are using,
-  // Since for the computation, we only need address_size.
-  // and address size are same between all the rnglists in .debug_rnglists.
+  // Reading rnglists from .debug_rnglists when the format was DW_FORM_sec_offset.
+  // Since we are calculating the address with DW_FORM_sec_offset, 
+  //  we don't care about (or need) the offset here. 
+  //  Therefore, I am getting the first element (rngdatamap_.begin())
+  //  and get the address_size from that element. 
+  //  Since the address size is equal between all elements.
```

Comment:
This is still confusing. You are saying `we don't care about (or need) the offset here` but you are using `offset` variable. I assume you are talking about another offset here. Also, please use either `we` or `I` consistently. The last sentence should be part of another sentence, it's not a proper sentence on its own.